common/app/model/content.scala

```diff
@@ -233,7 +237,52 @@ object Content {
     }
   }
 
-  def make(apiContent: contentapi.Content): Content = {
+  object BodyBlocks {
+    def make(blocks: Option[Blocks]): Seq[BodyBlocks] =
+    blocks.toSeq.flatMap { blocks =>
```

Comment:
There is a parameter to `make` called `blocks` of type `Option[Blocks]`, but then inside the flatMap, there's also the parameter of the anonymous function called `blocks` of type `Blocks` which hides the `Option[Blocks]` one from the inner scope.